yarn-project/prover-node/src/monitors/epoch-monitor.ts

```diff
@@ -47,15 +77,121 @@ export class EpochMonitor implements Traceable {
     if (!this.latestEpochNumber) {
       const epochNumber = await this.l2BlockSource.getL2EpochNumber();
       if (epochNumber > 0n) {
-        await this.handler?.handleEpochCompleted(epochNumber - 1n);
+        await this.addNewEpoch(epochNumber - 1n);
```

Comment:
Doesn't this mean we won't start proving the last epoch on startup? If there's a single prover, it means a reset of the prover-node will always trigger a reorg.